### DISCUSSION TITLE
Merging another GPS track with crwData

### DISCUSSION THREAD

**Discussion by eunjng** [bmcclintock/momentuHMM#170] [Q&A] [answered]
Hi Brett and Th´eo :)
I'm using and studying your great package, momentuHMM.
Studying your example code, I have a question about crawlWrap function.

I have GPS data which has different recording intervals from 2h to 12h.
So to make them have an identical time interval (2h), I used crawlWrap.
and the result of crawlWrap just included tracks that are predicted through crawlWrap.

So I can only use the prepData to the above crwData.
Is there a method to merge crwData with another GPS track data that are unnecessary to predict and correct?

**Comment by bmcclintock** [comment, marked as answer]
Presumably you found the `momentuHMM::crawlMerge` function?
